Array/readme.md

```diff
@@ -35,3 +36,4 @@ To create an array, you need to known the length (or size) of the array in advan
  * Inverse of an array ----> [C++](Code/C++/inverse_of_an_array.cpp) | [Python](/Code/Python/inverseArray.py)
```

Comment:
can you please update the whole list, as J comes before K and Find comes above so on...

Reply:
I'm really sorry sir last time I just overlooked the whole list, now it's done'. @rish-singhal